src/components/DimensionMenu.js

```diff
@@ -186,8 +186,9 @@ export class DimensionMenu extends Component {
                                         }}
                                     >
                                         {i18n.t(
-                                            `${getAxisItemLabelPrefix()} ${getAxisName(
```

Comment:
This should also use [`interpolation`](https://www.i18next.com/translation-function/interpolation) instead of a string template literal.

Reply:
And actually there's no point calling `i18n.t` here at all since there's no static text to translate

Reply:
Valid point, i18n removed